master/internal/api_model.go

```diff
@@ -544,8 +539,7 @@ func (a *apiServer) DeleteModel(
 		return nil, err
 	}
 	holder := &modelv1.Model{}
-	err = a.m.db.QueryProto("delete_model", holder, currModel.Name, user.User.Id,
```

Comment:
question: don't we need to check with authzProvider?

Reply:
I'm checking if user `CanDeleteModel` on `line 537`. Is there more that needs to be authenticated?

Reply:
ah I couldn't tell from the GH diff view. it shouldn't need more than that